tests/Feature/TrendingThreadsTest.php

```diff
@@ -14,8 +14,9 @@ protected function setUp()
     {
         parent::setUp();
 
-        $this->trending = new Trending();
+	config(['cache.default' => 'array']);
```

Comment:
I think this will be set in phpunit.xml

Reply:
Yep... brain fart, I removed the explicit config set from the test.